kratos/python/add_containers_to_python.cpp

```diff
@@ -129,6 +129,39 @@ template< class TBinderType, typename TContainerType, typename TVariableType > v
 //         }
 //     }
 
+template <class TVariableType>
```

Comment:
this depends on the open PR right? is that solved?

Reply:
#2902 , not yet resolved